MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using MobileManager.Models.Devices;
+using MobileManager.Models.Devices.Interfaces;
+using MobileManager.SeleniumConfigs.DataSets.Interfaces;
+using DotLiquid;

```

Comment:
what is this and how it is used in this class?

Reply:
Templating system - https://github.com/dotliquid/dotliquid - .NET port of Liquid https://shopify.github.io/liquid/
I will add comments about this